packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -100,15 +100,15 @@ export function PanelChrome({
   const showStreaming = loadingState === LoadingState.Streaming && !isUsingDeprecatedLeftItems;
 
   const renderStatus = () => {
-    if (isUsingDeprecatedLeftItems) {
-      return <div className={cx(styles.rightAligned, styles.items)}>{itemsRenderer(leftItems, (item) => item)}</div>;
-    } else {
-      const showError = loadingState === LoadingState.Error || status?.message;
-      return showError ? (
+    const showError = loadingState === LoadingState.Error || status?.message;
+    if (!isUsingDeprecatedLeftItems && showError) {
```

Comment:
Don't think this !leftItems check here is needed, loadingState and status are new props so should this should not show for any old uses

Reply:
Yes, I have this in my to-do for a separate PR so as to not put irrelevant things in here 🙌.
We added it at first because the leftItems and new state positions were both on the far right of the header, so could not exist together.

Reply:
I moved the render of `leftItems` out of this `renderStatus()` function because they are on the right side of the header together with the `menu`, which is much easier if they are wrapped together with `margin-left: auto` under the _flex_ container that is the header.